src/ekf_state_est/src/kalman_filter/data_association.cpp

```diff
@@ -59,18 +59,20 @@ bool SimpleMaximumLikelihood::validate(const Eigen::Vector2f& predicted_measurem
   // file << "VALIDATION VALUE" << validation_value << "\n";
   // file << "V" << v << "\n";
   file.flush();
-  S.diagonal().array() += 1e-6;
-  float nd = validation_value + log(S.determinant());
+  S.diagonal().array() += (float) 1e-6;
```

Comment:
Why are we using C-style casts? Or is this something else?

Reply:
Isto pus temporariamente mas é para tirar